src/python/build_management/build_manager.py

```diff
@@ -1111,84 +1151,88 @@ def get_revisions_list(bucket_path, testcase=None):
   return revision_list
 
 
-def setup_trunk_build():
+def setup_trunk_build(bucket_paths_env_vars=DEFAULT_BUILD_BUCKET_PATH_ENV_VARS,
```

Comment:
side node: would be even better if we remove the dependency on global (env) vars being set in this function, and get callers to supply bucket paths, but can leave for future cleanup :)

Reply:
Good idea, will try in a follow up CL with a few other changes that are a bit risky.